### PR TITLE
Added space invaders by Manish-bitpirate

### PR DESCRIPTION
**Pull request by Manish-bitpirate**
## Description
A simple space shooter game made using vanilla js

## Add GIF/ScreenShot 
Screenshot 2021-10-28 222443
8

## Task Checklist
Put x in between the square brackets without any spaces to tick your Task.  Create a PR only after you complete all the tasks.
- [✅] You have created a new branch for contribution.
- [✅] Added README.md for your project in a specified folder.
- [✅] Add Demo in GIF or Screenshot form.
- [✅] You are Registered Officially for HacktoberFest 2021 as a Participant.
- [✅] Followed Contribution Guidelines.

## Issue 
 #168

### CODE REVIEW

**Review by w3bdesign — APPROVED** [review]

File: Space-Invaders-Js/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Space-Invaders
```
Comment (soumikbaithalu) [inline Space-Invaders-Js/README.md:1]: - Add Gif in `readme.md` 
- Add instruction in `readme.md`
- Add author's name in `readme.md`

File: Space-Invaders-Js/app.js [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+  }
+  switch(e.key) {
+    case 'ArrowUp':
+      laserId = setInterval(moveLaser, 100)
+  }
+}
+
+document.addEventListener('keydown', shoot)
+
+})
```
Comment (soumikbaithalu) [inline Space-Invaders-Js/app.js:137]: Add a new line at EOF.

File: Space-Invaders-Js/design.css [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+.boom {
+    background-color:rgb(255, 238, 4);
+}
+
+.laser {
+    background-color:rgb(255, 0, 0);
+    height: 5px;
+    width: 5px;
+}
```
Comment (soumikbaithalu) [inline Space-Invaders-Js/design.css:44]: Add a new line at EOF.

**Review by soumikbaithalu — CHANGES_REQUESTED** [review]
Minnor Changes

**Comment by Harshil-Jani** [comment]
@Manish-bitpirate Plz make the required changes by today if you want it to get merged for Hacktoberfest.

File: Space-Invaders-Js/README.md (hunk 1 above)
Comment (Manish-bitpirate) [reply Space-Invaders-Js/README.md:1]: hey there,
i added the requested changes.

**Review by Manish-bitpirate — COMMENTED** [review]
updated files

**Review by soumikbaithalu — APPROVED** [review]
LGTM 💯

**Pull request merged**